execute/aggregate_test.go

```diff
@@ -87,7 +88,7 @@ func TestAggregateTransformation_ProcessChunk(t *testing.T) {
 		t.Fatal("expected aggregate function to be invoked, but it was not")
 	}
 	// Memory should have been released since we did not retain the data.
-	mem.AssertSize(t, 0)
+	assert.Equal(t, 0, mem.CurrentAlloc(), "unexpected memory allocation.")
```

Comment:
Did you need to make this change? I would expect anything where we're checking for `0` like in `AssertSize(t, 0)` to still pass, since there should be no outstanding allocations. Is it possible that this was actually finding a bug?